api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -137,13 +139,10 @@ public RosetteAPI(String key) throws IOException, RosetteAPIException {
      * @throws IOException         General IO exception
      */
     public RosetteAPI(String key, String alternateUrl) throws IOException, RosetteAPIException {
```

Comment:
I don't think that it would help. If you read the doc for [java.net.URL](https://docs.oracle.com/javase/7/docs/api/java/net/URL.html) the only validation that it performs is of the protocol. Given that and the fact we perform `String` manipulation it would make the most sense to leave it as is.